src/storage-client/src/controller.rs

```diff
@@ -1672,10 +1670,14 @@ where
             _ => unreachable!("verified collection refers to ingestion"),
         };
 
+        // Assess dependency since, which we have to fast-forward this
+        // collection's since to.
         let storage_dependencies = collection.description.get_storage_dependencies();
 
-        // Install read capability for all dependencies on new source exports.
-        self.install_dependency_read_holds(new_source_exports.into_iter(), &storage_dependencies)?;
+        // Install read capability for all dependencies on new source exports,
+        // which we can do now that we've fast-forwarded their sinces.
```

Comment:
When you say "now that we've fast-forwarded their sinces", it's a bit confusing to me because the method that's about to be called is meant to do that, right?